doc/network-zones.md

```diff
@@ -39,6 +34,11 @@ network.nat         | bool       | no       | true    | Whether to generate reco
 
 Additionally the `user.` key namespace is also supported for user-provided free-form key/value.
 
+This is controlled through `dns.zone.forward`, `dns.zone.reverse.ipv4` and `dns.zone.reverse.ipv6` in network configuration.
```

Comment:
Now that you've moved this line away from the `Each network can be related to up to 3 zones for:` statement above it has lost its context and so "this" doesn't mean a lot at this location in the document (now that its in the "network zone properties" section).

Did you move it because you don't want to have setting specific sentences in the intro section?

If so, then suggest rephrasing this line to something like:

```
Networks can be related to specific zones through the `dns.zone.forward`, `dns.zone.reverse.ipv4` and `dns.zone.reverse.ipv6` settings in their respective network configuration.
```

Reply:
Oh I see you added a heading in the next commit. Maybe worth including new headings when re-ordering if you intend to add it to allow the reviewer to see the proposed re-ordering in the context of the new headings, as we tend to review PRs using a commit by commit approach :)

Reply:
> Maybe worth including new headings when re-ordering if you intend to add it to allow the reviewer to see the proposed re-ordering in the context of the new headings, as we tend to review PRs using a commit by commit approach :)

For this PR, it's really only the last commit that has the relevant changes. Everything else is to make it easier to review ...

Reply:
Yeah makes sense now, my first comment was when looking at an earlier commit that didn't include the new heading and so "this" didn't make sense in the new context. With the new heading its fine.